src/components/bs5/inpagenav/inpagenav.stories.js

```diff
@@ -27,7 +27,7 @@ export const Default = {
       <div class="container-fluid">
         <div class="row">
           <div class="col-12">
-            <div class="">
+            <div class="qld-content-body">
```

Comment:
can it be updated to ``<div class="qld-content-body" id="content">``
as its usually wrapped in the content id also

Reply:
Added in via further commit